File: 026/RemoveDuplicatesFromSortedArray.java
public class RemoveDuplicatesfromSortedArray {
    public int removeDuplicates(int[] nums) {
        if (nums.length == 0)
            return 0;
        int val = nums[0], size = 1;
        for (int i : nums)
            if (i != val)
                nums[size++] = val = i;
        return size;
    }
}

File: 026/RemoveDuplicatesFromSortedArray.kt
class RemoveDuplicatesfromSortedArray {
 	fun removeDuplicates(nums: IntArray): Int {
 		if (nums.size == 0)
 			return 0
 		var value = nums[0]
 		var size = 1
 		for (i in nums)
 			if (i != value) {
 				value = i
 				nums[size++] = value
 			}
 		return size
 	}
}

fun main(args: Array<String>) {
    var nums: IntArray = intArrayOf(0,0,1,1,1,2,2,3,3,4)
    println(RemoveDuplicatesfromSortedArray().removeDuplicates(nums))
}